### PR TITLE
Enable the new test framework to test across the new function

### PR DESCRIPTION
**Pull request by munroesj52**
added for the new header files.

2018-06-12 Steven Munroe <munroesj52@gmail.com>

	* src/vec_char_ppc.h (vec_isalnum, vec_isalpha, vec_isdigit,
	vec_toupper, vec_tolower): Format for GNU.

	* src/testsuite/arith128_print.c
	(print_vint32d, print_vint32x, print_vint64d, print_vint64x):
	New functions.
	(print_vint8c, print_vint8x, print_vint8d): New functions.
	(print_vint256): Correct for Endian.
	(checK_vui8_priv): New function.
	(db_shift_rightq, db_shift_leftq, db_vec_clzq, db_vec_mulluq,
	db_vec_muludq, db_vec_addeuqm, db_vec_addeuqm, db_vec_mul10uq
	db_vec_mul10cuq):
	Moved to src/testsuite/arith128_test_i128.c.
	(db_vec_isalpha, db_vec_toupper, db_vec_tolower):
	Moved to src/testsuite/arith128_test_char.c.
	(db_sdiv_qrnnd): Removed.
	* src/testsuite/arith128_print.h
	(print_vint32d, print_vint32x, print_vint64d, print_vint64x):
	New functions.
	(print_vint8c, print_vint8x, print_vint8d): New functions.
	(checK_vui8, checK_vui8_priv): New function.
	(db_shift_rightq, db_shift_leftq, db_vec_clzq, db_vec_mulluq,
	db_vec_muludq, db_vec_addeuqm, db_vec_addeuqm, db_vec_mul10uq
	db_vec_mul10cuq):
	Moved to src/testsuite/arith128_test_i128.h.
	(db_vec_isalpha, db_vec_toupper, db_vec_tolower):
	Moved to src/testsuite/arith128_test_char.h.
	* src/testsuite/arith128_test_i128.c (db_shift_rightq,
	db_shift_leftq, db_vec_clzq, db_vec_mulluq, db_vec_muludq,
	db_vec_addeuqm, db_vec_addeuqm, db_vec_mul10uq,
	db_vec_mul10cuq): New functions.
	Moved from src/testsuite/arith128_print.c.
	(test_1): Removed.
	(test_2): Renamed test_addq.
	(test_3): Movec to src/testsuite/arith128_test_bcd.c.
	(Test_4): Renamed test_mul10uq.
	(Test_4b): Renamed test_mul10ecuq.
	(test_4b1): Removed.
	(Test_4c): Renamed test_cmul100.
	(Test_5): Renamed test_mulluq.
	(Test_6): Renamed test_muludq.
	(Test_8): Renamed test_clzq.
	(test_43): Removed.
	(test_44): Renamed test_vsrqi.
	(test_45): Renamed test_vslqi.
	(test_46): Renamed test_vsrq.
	(test_47): Renamed test_vslq.
	(test_msumudm, test_revbq, test_popcntq): New Function.
	(test_vec_i128): New Function.
	* src/testsuite/arith128_test_i128.h (db_shift_rightq,
	db_shift_leftq, db_vec_clzq, db_vec_mulluq, db_vec_muludq,
	db_vec_addeuqm, db_vec_addeuqm, db_vec_mul10uq,
	db_vec_mul10cuq): New functions.
	(test_1): Removed.
	(test_2): Renamed test_addq.
	(test_3): Removed.
	(Test_4): Renamed test_mul10uq.
	(Test_4b): Renamed test_mul10ecuq.
	(Test_4c): Renamed test_cmul100.
	(Test_5): Renamed test_mulluq.
	(Test_6): Renamed test_muludq.
	(test_8): Removed.
	(test_43): Removed.
	(test_44): Removed.
	(test_45): Removed.
	(test_46): Removed.
	(test_47): Removed.
	(test_msumudm, test_vec_i128): New Function.

	* src/testsuite/pveclib_test.c (Main): Add calles to
	test_vec_char, test-vec_bcd, test_vec_i16, test_vec_i32,
	test_vec_i64, test_vec_i128.
	Remove calls to test_2, test_3, test_4, test_4b, test_4c,
	test_5, test_6, test_7, test_8, test_43, test_44, test_45,
	test_46, test_47.

	* src/Makefile.am (pveclibinclude_HEADERS):
	Add vec_int64_ppc.h, vec_int32_ppc.h. vec_int16_ppc.h,
	vec_char_ppc.h, vec_bcd_ppc.h.
	(pveclib_test_SOURCES): Add testsuite/arith128_test_i64.c,
	testsuite/arith128_test_i32.c, testsuite/arith128_test_i16.c,
	testsuite/arith128_test_char.c, testsuite/arith128_test_bcd.c.

Signed-off-by: Steven Munroe <munroesj52@gmail.com>

### CODE REVIEW

File: src/Makefile.am [hunk 1]
```diff
@@ -38,7 +43,14 @@ TESTS += pveclib_test
...
+	testsuite/arith128_test_i64.c \
+	testsuite/arith128_test_i32.c \
+	testsuite/arith128_test_i16.c \
+	testsuite/arith128_test_char.c \
+	testsuite/arith128_test_bcd.c
+	
 pveclib_test_LDADD = .libs/libpvec.a
 
 check_PROGRAMS = $(TESTS)
+
```
Comment (ThinkOpenly) [inline src/Makefile.am]: nit: extra blank line added

File: src/testsuite/arith128_print.c [hunk 2]
```diff
@@ -391,21 +417,49 @@ print_vint8 (char *prefix, vui8_t val)
...
+}
+
 void
 print_vchar (char *prefix, vui8_t val)
 {
   printf ("%s%c%c%c%c%c%c%c%c%c%c%c%c%c%c%c%c\n", prefix, val[0], val[1],
-          val[2], val[3], val[4], val[5], val[6], val[7], val[8], val[9],
-          val[10], val[11], val[12], val[13], val[14], val[15]);
+	  val[2], val[3], val[4], val[5], val[6], val[7], val[8], val[9],
+	  val[10], val[11], val[12], val[13], val[14], val[15]);
```
Comment (ThinkOpenly) [inline src/testsuite/arith128_print.c]: what changed here? some sort of whitespace modification? (I know, annoying.)

File: src/testsuite/arith128_print.c [hunk 3]
```diff
@@ -414,9 +468,9 @@ print_vbool8 (char *prefix, vui8_t val)
   merge = vec_or (tpart, fpart);
 
   printf ("%s%c%c%c%c%c%c%c%c%c%c%c%c%c%c%c%c\n", prefix, merge[0], merge[1],
-          merge[2], merge[3], merge[4], merge[5], merge[6], merge[7], merge[8],
-          merge[9], merge[10], merge[11], merge[12], merge[13], merge[14],
-          merge[15]);
+	  merge[2], merge[3], merge[4], merge[5], merge[6], merge[7], merge[8],
```
Comment (ThinkOpenly) [inline src/testsuite/arith128_print.c]: more whitespacey

**Comment by munroesj52** [comment]
Thanks

**Pull request merged**